prow/scripts/build-generic.sh

```diff
@@ -0,0 +1,47 @@
+#!/usr/bin/env bash
+
+set -e
+
+readonly SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+# shellcheck disable=SC1090
+source "${SCRIPT_DIR}/library.sh"
+
+usage () {
+    echo "Usage: \$ ${BASH_SOURCE[1]} /path/to/component"
+    exit 1
+}
+
+readonly SOURCES_DIR=$1
+
+if [[ -z "${SOURCES_DIR}" ]]; then
+    usage
+fi
+
+init
+
+if [ -n "${PULL_NUMBER}" ]; then
+  echo "Building from PR"
+  DOCKER_TAG="PR-${PULL_NUMBER}"
+elif [[ "${PULL_BASE_REF}" =~ release-.* ]]; then
+  echo "Building from release ${PULL_BASE_REF}"
+  DOCKER_TAG=$(cat "${SCRIPT_DIR}/../../RELEASE_VERSION")
```

Comment:
Wouldn't this be `$SOURCES_DIR` ?

Reply:
Not yet. RELEASE_VERSION is in test-infra now.